examples/README.md

```diff
@@ -3409,12 +3509,52 @@ And then our reusable sub-workflow which performs the checking of our car vitals
       },
       {
          "name": "WaitTwoMinutes",
-         "type": "delay",
-         "timeDelay": "PT2M",
-         "end": true
+         "type": "event",
+         "timeout": "PT2M",
+         "onEvents": [
+            {
+              "eventRefs": ["StopVitalsCheck"],
+              "eventDataFilter": {
+                "data": "${ true }",
```

Comment:
What is the ${ true } expression supposed to do?

Reply:
It's supposed to replace the results of the event, so that the value assigned to .stopReceived is 'true'.

Reply:
Switched to using has() in the switch state's data-condition.